app/src/main/res/values/component_colors.xml

```diff
@@ -36,4 +36,10 @@
   <color name="admin_pin_activity_layout_background_color">@color/background_color</color>
   <!-- Pin Password Activity -->
   <color name="pin_password_activity_layout_background_color">@color/background_color</color>
+<!--  Topic Activity-->
+  <color name="topic_activity_layout_background_color">@color/background_color</color>
+  <color name="topic_lessons_story_summary_card_background_color">@color/container_background_color</color>
+  <color name="topic_practice_subtitle_text_color">@color/secondary_text_color</color>
+  <color name="topic_info_thumbnail_gradient_end">@color/background_color</color>
+  <color name="topic_activity_tab_layout_color">@color/tab_bar_color</color>
```

Comment:
There has been a change to the decided convention. We decided to keep the file names in which the colors are declared as a prefix. Refer to this [comment](https://github.com/oppia/oppia-android/pull/4205#issuecomment-1052251992) for details.

Reply:
Fixed that